Product Engineer Handbook (#14530)
PostHog/posthog.com: contents/product-engineer-handbook/skills.md

---
title: Things product engineers need to know
showTitle: true
---

Product engineers are T or M-shaped people – i.e. engineers who have deep knowledge and experience in one or more areas, and broad knowledge of several others.

> Some will be better at [talking to users](/newsletter/talk-to-users) than others. Some will be stronger at data analysis. Some will better on the front-end than the back-end.

Reviewer: less words
```suggestion
Some are better at [talking to users](/newsletter/talk-to-users) than others. Some are stronger at data analysis. Some are better on the front-end than the back-end.
```
